research-hub-web/src/app/components/search-results/search-filters/search-filters.component.ts

```diff
@@ -1,22 +1,58 @@
 import { Component, Input, OnDestroy, OnInit, ViewEncapsulation } from '@angular/core';
 import { FormGroup } from '@angular/forms';
-import { Subscription, Observable, forkJoin } from 'rxjs';
+import { Subscription, forkJoin } from 'rxjs';
 import { Tag } from '../mat-tags/mat-tags.component';
-import { OptionsService, RoleTypeId } from '@services/options.service';
 import { ResearchHubApiService, PeopleParams, Params } from '@services/research-hub-api.service';
 import { SearchResultsComponent } from '../search-results.component';
 import { ListItem } from '@model/ListItem';
 import { OrgUnit } from '@model/OrgUnit';
 import { AnalyticsService } from '@services/analytics.service';
 
+enum RoleTypeId {
+  ServiceOwner = 1,
+  CopyOwner,
+  UserSupport
+}
+
+enum OptionType {
+  ResearchActivity = 1,
+  Category,
+  Menu
+}
+
+enum CategoryId {
+  All = 1,
+  Support,
+  Equipment,
+  Training,
+  Software,
+  Facilities,
+  Guide,
+  Person,
+  Policies,
+  Articles,
+  SubHubs
+}
+
+enum ContentTypeId {
+  Support = 1,
+  Equipment,
+  Training,
+  Software,
+  Facilities,
+  KnowledgeArticle,
+  Guide
+}
+
 @Component({
   selector: 'app-search-filters',
   templateUrl: './search-filters.component.html',
   styleUrls: ['./search-filters.component.scss'],
   encapsulation: ViewEncapsulation.None
 })
 export class SearchFiltersComponent implements OnInit, OnDestroy {
-
+  public categoryOptions: any[];
+  public contentTypeMap: any;
```

Comment:
Again avoid `any`, but I'm going to be refactoring all this soon so you can leave it